app/assets/javascripts/premadeModal.js

```diff
@@ -0,0 +1,22 @@
+var onReadyStrategiesModal = function() {
+
+	var modal = document.getElementById('myModal');
+	var btn = document.getElementById("myBtn");
+	var span = document.getElementsByClassName("close")[0];
+	
+	if(modal || btn || span){
+		btn.onclick = function() {
+		    modal.style.display = "block";
```

Comment:
What happens if the modal is already open and you end up here (e.g. clicking btn super quickly)? Think we should chick if the modal is not present, then display it.